fault/system_verilog_target.py

```diff
@@ -0,0 +1,191 @@
+from fault.verilog_target import VerilogTarget
+import magma as m
+from pathlib import Path
+import fault.actions as actions
+import fault.verilator_utils as verilator_utils
+from bit_vector import BitVector
+import fault.value_utils as value_utils
+import subprocess
+from fault.util import flatten
+
+
+src_tpl = """\
+`timescale 1ns/1ns
```

Comment:
How important is this parameter? Does this need to be a parameter? Maybe @alexcarsello can help here.

Reply:
We can make `timescale` a parameter to the target, so have this by default for most use cases, but allow the user to set it.